core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationUtils.java

```diff
@@ -429,20 +431,37 @@ public static boolean autoReloadConfig(IbisContext ibisContext, String name, Str
 
 	/**
 	 * 
-	 * @param ibisContext
-	 * @return A map with all configurations to load (KEY = configName, VALUE = ClassLoader)
+	 * @return A map with all configurations to load (KEY = ConfigurationName, VALUE = ClassLoaderType)
 	 */
 	public static Map<String, String> retrieveAllConfigNames(IbisContext ibisContext) {
-		// For now only database configurations are returned, but also
-		// configuration from other resources (like file system directories) can
-		// be added
 		Map<String, String> allConfigNameItems = new LinkedHashMap<String, String>();
 
 		StringTokenizer tokenizer = new StringTokenizer(CONFIGURATIONS, ",");
 		while (tokenizer.hasMoreTokens()) {
 			allConfigNameItems.put(tokenizer.nextToken(), null);
 		}
 
+		if (CONFIG_AUTO_FS_CLASSLOADER) {
+			try {
+				String configDir = AppConstants.getInstance().getProperty("configurations.directory");
+				if(StringUtils.isEmpty(configDir))
+					throw new IOException("property [configurations.directory] not set");
```

Comment:
Waarom twee properties? Je kan toch gewoon kijken of "configurations.directory" niet leeg is, en dan gaan laden?